development/java/Rubberduck.Parsing/Grammar/build.gradle

```diff
@@ -0,0 +1,52 @@
+buildscript {
+    repositories {
+        mavenCentral();
+    }
+
+    dependencies {
+        classpath group: 'org.antlr', name: 'antlr4', version: '4.7.2'
```

Comment:
we possibly need to remember to keep the ANTLR version in sync between our c# code and the java code.

Reply:
@Vogel612

I've just noticed 4.6.4 is in used, so will modify that manually.
Is there a easy to read place where the desired version number could be extracted from in the build script?

Reply:
no, not even remotely. There might be a way to do some XML-parsing on a specific project file (because nuget packages are defined in the csproj files), but I don't think we can really use that.

Reply:
@Vogel612 While it is possible, I would say this is out of scope, would you agree?

Wierdly, after manually changing the antlr version to 4.6, the generated code tries to override non-existant method getChannelNames, so I have left it at 4.7.2. Is this OK?

Reply:
Yes I would very much agree that this is out of scope. Relatedly we should be looking into upgrading our ANTLR dependency to the latest version anyways. Compare also [chat](https://chat.stackexchange.com/transcript/message/49154845#49154845)

Reply:
Note that this is [Sam Harwell's fork](https://github.com/sharwell/antlr4cs), which remove the need to build the ANTLR using Java. It look like the latest is 4.6.5, even though the official ANTLR is further along.